### PR TITLE
Fixes #23731: Debian source list should force architecture

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/23731

### CODE REVIEW

File: src/reference/modules/installation/pages/_partials/debian_repo.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ Add Rudder's package repository:
 #   stretch for Debian 9
 #   bionic  for Ubuntu 18.04 LTS
 
-echo "deb http://repository.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
+echo "deb [arch=amd64] http://repository.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
```
Comment (amousset) [inline src/reference/modules/installation/pages/_partials/debian_repo.adoc:11]: this will not work on arm systems

Comment (peckpeck) [reply src/reference/modules/installation/pages/_partials/debian_repo.adoc:11]: there is no arm system on public repository

Comment (amousset) [reply src/reference/modules/installation/pages/_partials/debian_repo.adoc:11]: indeed

File: src/reference/modules/installation/pages/_partials/debian_repo.adoc [hunk 2]
```diff
@@ -21,7 +21,9 @@ You need to replace the user name (LOGIN) and the password (PASSWORD) by your Ru
 [source, Bash]
 ----
 
-echo "deb https://download.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
+# arch may be replaced by i386, armhf or arm64 depending on your subscription and actual architecture
+
+echo "deb [arch=amd64] https://download.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
```
Comment (amousset) [inline src/reference/modules/installation/pages/_partials/debian_repo.adoc:26]: this will not work on arm systems

Comment (amousset) [reply src/reference/modules/installation/pages/_partials/debian_repo.adoc:26]: (this time on the right line)

Comment (peckpeck) [reply src/reference/modules/installation/pages/_partials/debian_repo.adoc:26]: there is a comment just above to explain that

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**